src/seedu/addressbook/parser/Parser.java

```diff
@@ -73,33 +59,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
+                return prepareAdd(arguments);
+
+            case AddMultipleCommand.COMMAND_WORD:
+                return new AddMultipleCommand(Integer.parseInt(arguments.replaceAll("\\s", "")));
 
-        case DeleteCommand.COMMAND_WORD:
-            return prepareDelete(arguments);
+            case DeleteCommand.COMMAND_WORD:
```

Comment:
Note the [coding style requirements](https://nus-cs2103-ay1819s2.github.io/cs2103-website/coding-standards/java/intermediate.html) for switch-case statements - there shouldn't be indentations for case clauses. It would be helpful to configure your IDE accordingly.